### DISCUSSION TITLE
Any tips for compatibility with Steam Deck?

### DISCUSSION THREAD

**Discussion by ThatWayOne** [SnosMe/awakened-poe-trade#896] [Q&A]
I haven't found any working solutions to use awakened Poe trade on the steam deck, and I was wondering if anyone here might have tips for getting this working? Thanks

**Comment by shyney7** [comment]
I would also like to know this. I first thought it would be as easy as just using the AppImage and binding `Ctrl + D` to a button in the Steam Controller Layout. My guess is that there is some path mismatch in the executable that could be easily fixed!?

@ThatWayOne Did you try to run the AppImage from the Terminal?
In the Terminal navigate to the same path as the AppImage with `cd` command and make the AppImage executable with `chmod +x ./Awakened-Trade.appimage` and than run `./Awakened-Trade.appimage`

**Comment by sfedu-admin** [comment]
Working flawlessly on Steam Deck in Desktop Mode.
Have tried to install [AppImageLauncher](https://github.com/TheAssassin/AppImageLauncher) and add Awakened PoE Trade to Game mode.
No luck here.

**Comment by bruno-fs** [comment]
I made it work on desktop mode only, but had to run poe in windowed mode. with that, I bound ctrl+D to one of the back buttons and it worked. I also set the right trackpad as a mouse so I could click back on the game.

Oddly enough, if the game is on fullscreen or windowed fullscreen, awakened poe trade never appears. If I hit shift+space on this mode, awakened poe trade appears, but doesn't show the game as its background (something that happens on windowed mode. My guess is that gamescope kicks in when the game is fullscreen and APT is incompatible with it.

Had no luck with gamemode, but I'm thinking a viable workaround could be running it with `--no-overlay` mode and use steam overlay's browser to access it.
